tools/bump_version.js

```diff
@@ -0,0 +1,58 @@
+const { readFileSync, writeFileSync } = require('fs');
+const { join } = require('path');
+const semver = require('semver')
+const { execSync } = require("child_process");
+
+const filesPaths = [
+    // CSharp
+    "csharp/Svix/Svix.csproj",
+    // Go
+    "go/internal/version/version.go",
+    // Java
+    "java/gradle.properties",
+    "java/README.md",
+    "java/lib/src/main/java/com/svix/Svix.java",
+    // Javascript
+    "javascript/package.json",
+    "javascript/src/index.ts",
+    // Kotlin
+    "kotlin/gradle.properties",
+    "kotlin/README.md",
+    "kotlin/lib/src/main/kotlin/SvixOptions.kt",
+    // Python
+    "python/svix/__init__.py",
+    // Ruby
+    "ruby/Gemfile.lock",
+    "ruby/lib/svix/version.rb"
+];
+
+const rootDir = join(__dirname, "..");
+
+if (process.argv.length !== 3 || !semver.valid(process.argv[2])) {
+    console.error("must supply a valid semantic version number");
+    return;
+}
+const newVersion = process.argv[2];
+const currentVersion = execSync("git describe --abbrev=0 --tags").toString().trim().replace("v", "");
+
+if (semver.lt(newVersion, currentVersion)) {
```

Comment:
Should also complain if they are equal, right?

Reply:
yep, added